### PR TITLE
bpo-47061: deprecate the `aifc` module

### PR DESCRIPTION
**Pull request by brettcannon**
Also introduce `warning._deprecated()` and `test.support.warnings_helper.import_deprecated()`.

https://bugs.python.org/issue47061

### CODE REVIEW

File: Lib/warnings.py [hunk 1]
```diff
@@ -483,6 +483,27 @@ def __exit__(self, *exc_info):
         self._module._showwarnmsg_impl = self._showwarnmsg_impl
 
 
+_DEPRECATED_MSG = "{name!r} is deprecated and slated for removal in Python {remove}"
```
Comment (tiran) [inline Lib/warnings.py:486]:
```suggestion
_DEPRECATED_MSG = "{name!r} is deprecated and slated for removal in Python {remove}."
```

Sorry, I could not resist. :)

File: Lib/warnings.py [hunk 2]
```diff
@@ -483,6 +483,27 @@ def __exit__(self, *exc_info):
...
+    RuntimeError is raised if *remove* specifies a major/minor tuple older than
+    the current Python version or the same version but past the alpha.
+
+    The *message* argument is formatted with *name* and *remove* as a Python
+    version (e.g. "3.11").
+
+    """
+    remove_formatted = f"{remove[0]}.{remove[1]}"
+    if (_version[:2] > remove) or (_version[:2] == remove and _version[3] != "alpha"):
+        msg = f"{name!r} was slated for removal after Python {remove_formatted} alpha"
```
Comment (tiran) [inline Lib/warnings.py:500]:
```suggestion
        msg = f"{name!r} was slated for removal after Python {remove_formatted} alpha."
```

File: Lib/warnings.py (hunk 1 above)
Comment (brettcannon) [reply Lib/warnings.py:486]: I could find anything in PEP 8 about ending exception/warning messages with a period and the code base is inconsistent.

Comment (tiran) [reply Lib/warnings.py:486]: Never mind. Guido pointed out that most exception messages don't have a full stop, too. Let's stick to your approach.

**Review by tiran — APPROVED** [review]

**Pull request merged**